tests/test_manager_pytest.py

```diff
@@ -1,13 +1,17 @@
 import sys
 import os
+
+from simulator import Simulator
+
 sys.path.append(os.path.dirname(os.path.abspath(__file__)) + "/..")
 from backend.manager import Manager
 from backend.producer import Producer
 from .mock_clock import MockClock
 
 def test_register_producer():
     # Make a manager with no neighbourhood (not needed for testing)
-    manager = Manager()
+    sim = Simulator()
```

Comment:
the tests now need a `Simulator()` in order to have a `clock.now` for contract id